.github/workflows/stale-bot.yaml

```
@@ -0,0 +1,21 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
```

For reproducibility of the workflows, it's better to pin the runtime as there could be sudden breaking changes:
```suggestion
    runs-on: ubuntu-22.04
```